src/Juvix/Compiler/Pipeline/Root/Base.hs

```diff
@@ -0,0 +1,15 @@
+module Juvix.Compiler.Pipeline.Root.Base where
+
+import Juvix.Compiler.Pipeline.Package.Base
+import Juvix.Prelude
+
+data Roots = Roots
```

Comment:
Is it better to call this one simply `Root`?

Reply:
Yes I think this is a better name. But would you mind if we did not change this until the other PRs are merged? It will cause many conflicts.

Reply:
Yes, I approved it because of the same reason.